src/aircraft_cmd.cpp

```diff
@@ -635,6 +635,12 @@ static int UpdateAircraftSpeed(Aircraft *v, uint speed_limit = SPEED_LIMIT_NONE,
 	 * and take-off speeds being too low. */
 	speed_limit *= _settings_game.vehicle.plane_speed;
 
+	/* adjust speed for broken vehicles */
+	if (v->vehstatus & VS_AIRCRAFT_BROKEN) {
+		if (speed_limit > SPEED_LIMIT_BROKEN) hard_limit = false;
```

Comment:
Can you please reverse the condition:
if (SPEED_LIMIT_BROKEN < speed_limit) hard_limit = false;

That way the code is symmetric to the code below ("new limit < old limit"), which makes the code is easier to read.